packages/next/src/client/app-webpack.ts

```diff
@@ -5,6 +5,13 @@ import { getDeploymentIdQueryOrEmptyString } from '../build/deployment-id'
 
 declare const __webpack_require__: any
 
+function encodeFile(file: string) {
```

Comment:
is there a reason we need the definition of encodeFile repeated 4 times across different places?

- packages/next/src/client/app-webpack.ts
- packages/next/src/client/route-loader.ts
- packages/next/src/pages/_document.tsxpackages/next/src/server/app-render/create-component-styles-and-scripts.tsx
- packages/next/src/server/app-render/get-layer-assets.tsx

feels like this could easily get out of sync if someone goes and changes it further. i know i'd expect "Find references" to show me all the places that use it, but this'd break that expectation in a pretty surprising way